runtime/syntax/typescript.yaml

```diff
@@ -1,7 +1,7 @@
 filetype: typescript
 
 detect:
-    filename: "\\.ts$"
+    filename: "\\.(ts|tsx)$"
```

Comment:
This can be shortened to just: `\\.tsx?$`.

Reply:
Sweet! I just did that.